backend/sokdak/src/test/java/com/wooteco/sokdak/member/service/MemberServiceTest.java

```diff
@@ -51,4 +57,32 @@ void signUp() {
         assertThat(memberRepository.findByUsernameValueAndPasswordValue(Encryptor.encrypt("josh"),
                 Encryptor.encrypt("Abcd123!@"))).isPresent();
     }
+
+    @DisplayName("닉네임 수정 기능")
+    @Test
+    void editNickname() {
+        NicknameUpdateRequest nicknameUpdateRequest = new NicknameUpdateRequest("chrisNick2");
+        memberService.editNickname(nicknameUpdateRequest, AUTH_INFO);
+        Member member = memberRepository.findById(AUTH_INFO.getId()).orElseThrow();
+
+        assertThat(member.getNickname()).isEqualTo("chrisNick2");
```

Comment:
```suggestion
        NicknameUpdateRequest nicknameUpdateRequest = new NicknameUpdateRequest("chrisNick2");
        
        memberService.editNickname(nicknameUpdateRequest, AUTH_INFO);
        
        Member member = memberRepository.findById(AUTH_INFO.getId()).orElseThrow();
        assertThat(member.getNickname()).isEqualTo("chrisNick2");
```
개행으로 when~then~given을 분리해주면 좋을 것 같아요!

Reply:
수정했습니다!